docker-compose.yml-example

```diff
@@ -215,18 +201,42 @@ services:
       - web-net
       - kopano-net
 
+  kopano_grapi:
+    image: ${docker_repo:?err}/kopano_core:${CORE_VERSION}
+    container_name: kopano_grapi
+    volumes:
+      - kopanosocket/:/run/kopano
+    environment:
+      - SERVICE_TO_START=grapi
+      - TZ=${TZ}
+    networks:
+      - kopano-net
+
+  kopano_kapi:
+    image: ${docker_repo:?err}/kopano_core:${CORE_VERSION}
+    container_name: kopano_kapi
+    volumes:
+      - kopanossl/:/kopano/ssl
+      - kopanosocket/:/run/kopano
+    environment:
+      - SERVICE_TO_START=kapid
+      - TZ=${TZ}
+      - KCCONF_KAPID_LOG_LEVEL=DEBUG
+      - KCCONF_KAPID_OIDC_ISSUER_IDENTIFIER=http://kopano_konnect:8777 # if this works, then it should go in the python script
```

Comment:
While this works to load the discovery document, it ultimatively still will not work when the iss as configured within konnect is not resolvable by kapid since the discovery document contains links which need to be loaded (jwks for example).

Reply:
Yes, I have already realized that. This will need to go back to $fqdn.